src/main/kotlin/com/atlassian/performance/tools/report/api/judge/ActionReportImpl.kt

```diff
@@ -0,0 +1,18 @@
+package com.atlassian.performance.tools.report.api.judge
+
+import com.atlassian.performance.tools.jiraactions.api.ActionType
+import com.atlassian.performance.tools.report.api.junit.JUnitReport
+import java.nio.file.Path
+
+internal class ActionReportImpl(
```

Comment:
IMO `*Impl` means you don't know how to name the implementation. If it's true, then why do we need the SPI?